mlflow/models/dependencies_schemas.py

```diff
@@ -66,18 +70,18 @@ def _get_retriever_schema():
     Returns:
         VectorSearchIndex: The vector search index schema.
     """
-    if not globals().get(DATABRICKS_RETRIEVER_PRIMARY_KEY, None) or not globals().get(
-        DATABRICKS_RETRIEVER_TEXT_COLUMN, None
+    if not globals().get(_RETRIEVER_PRIMARY_KEY, None) or not globals().get(
```

Comment:
nit: we can convert all `get(xxx, None)` to `get(xxx)` as it by default returns None if non-existing